.zenodo.json

```diff
@@ -1,40 +1,92 @@
 {
-  "creators": [
-    {
-      "name": "DuPre, Elizabeth",
-      "affiliation": "Montreal Neurological Institute, McGill University",
-      "orcid": "0000-0003-1358-196X"
-    },
-    {
-      "name": "Salo, Taylor",
-      "affiliation": "Florida International University",
-      "orcid": "0000-0001-9813-3167"
-    },
-    {
-      "name": "Markello, Ross",
-      "affiliation": "Montreal Neurological Institute, McGill University",
-      "orcid": "0000-0003-1057-1336"
-    },
-    {
-      "name": "Kundu, Prantik",
-      "affiliation": "Mount Sinai Hospital",
-      "orcid": "0000-0001-9367-3068"
-    },
-    {
-      "name": "Whitaker, Kirstie",
-      "affiliation": "Alan Turing Institute; Cambridge University",
-      "orcid": "0000-0001-8498-4059"
-    },
-    {
-      "name": "Handwerker, Dan",
-      "affiliation": "National Institutes of Health",
-      "orcid": "0000-0001-7261-4042"
-    }
-  ],
-  "keywords": [
-    "neuroimaging",
-    "fMRI"
-  ],
-  "license": "LGPL-2.1",
-  "upload_type": "software"
+    "creators": [
+        {
+            "name": "The tedana Community"
+        },
+        {
+            "name": "Caballero-Gaudes, César",
+            "affiliation": "Basque Center on Cognition, Brain and Language",
+            "orcid": "0000-0002-9068-5810"
+        },
+        {
+            "name": "Dowdle, Logan T.",
+            "affiliation": "Center for Magnetic Resonance Research, University of Minnesota",
+            "orcid": "0000-0002-1879-705X"
+        },
+        {
+            "name": "DuPre, Elizabeth",
+            "affiliation": "Montreal Neurological Institute, McGill University",
+            "orcid": "0000-0003-1358-196X"
+        },
+        {
+            "name": "Handwerker, Dan",
+            "affiliation": "National Institutes of Health",
+            "orcid": "0000-0001-7261-4042"
+        },
+        {
+            "name": "Heunis, Stephan",
+            "affiliation": "Eindhoven University of Technology",
+            "orcid": "0000-0003-3503-9872"
+        },
+        {
+            "name": "Kundu, Prantik",
+            "affiliation": "Mount Sinai Hospital",
```

Comment:
I'm actually not sure if he's still affiliated with Mount Sinai. I know that he's now a research head at [Hyperfine](https://hyperfine.io/), so we might want to update his affiliation ?

Reply:
Why do we even include affiliations when they change so much in academia? We've got ORCIDs and those are at least stable...

Reply:
His CV says he's still at Mount Sinai _and_ Hyperfine, so I think we're good.

Reply:
I cannot find his CV 😅 Is it on LinkedIn ? I always get locked out of LinkedIn profiles since I refuse to make an account....
But thanks for confirming ! Should we add Hyperfine, then ?

Reply:
Yes it's linked from his LinkedIn.

> Should we add Hyperfine, then ?

Do folks normally include private companies in their affiliations? If so, then I can do that.

Reply:
I want to say yes, but I'm having a hard time finding an immediate example. Though if you'd rather leave it off because of the institution-type, I'm fine with that as a guideline for now.

Reply:
If he was _just_ affiliated with Hyperfine, then I'd use that, but since he has an academic affiliation as well, I feel like that's enough. There's also an element of laziness, to be honest.